frontend_react/src/components/Navbar.jsx

```diff
@@ -14,26 +14,30 @@ function Navbar({ toggleTheme, theme }) {
 
   return (
     <nav
-      className="appBar flex flex-row justify-between items-center px-4 py-3 w-full backdrop-blur-sm backdrop-filter z-0 border-b-2 border-navbarborder md:border-transparent fixed"
+      className="appBar flex flex-row justify-between items-center px-4 py-3 w-full backdrop-blur-sm backdrop-filter z-50 border-b-2 border-navbarborder md:border-transparent fixed"
     >
       <a
         href="#HOME"
         className="logo flex justify-start items-center "
       >
         <img src={logo} alt="logo" className="flex-1 object-cover max-w-[70px] max-h-[70px] shadow-primary shadow-sm rounded-full" />
       </a>
-      <ul className="links hidden md:flex flex-1 justify-end items-center list-none text-secondary text-base">
-        {['HOME', 'ABOUT', 'PROJECTS', 'CONTACT', <Button btnName="RESUME" />, theme === 'light' ? <BsFillMoonStarsFill onClick={toggleTheme} size={20} /> : <BsSunFill onClick={toggleTheme} size={20} color="yellow" />].map((item) => (
-          <li key={`link - ${item}`} className="flex flex-col mx-4 cursor-pointer justify-center items-center ">
+      <ul className="links hidden md:flex flex-1 justify-end items-center list-none text-base">
+        {['HOME', 'ABOUT', 'PROJECTS', 'CONTACT', <a href="https://www.dropbox.com/s/5arl2s285thrzej/Richard_Leung%20Woo-Gabriel_Resume.pdf?dl=0" download="resume.pdf" target="_blank" rel="noreferrer">RESUME</a>,
+        ].map((item) => (
+          <li key={`${item}`} className="flex flex-col mx-4 cursor-pointer justify-center items-center ">
```

Comment:
You use `${variable_name}` if you want to concat a variable with a string or to enclose an expression that will be interpolated into a string. It is commonly referred to as "string interpolation."

Over here, you're not doing any of these. So key={item} would be right!